### PR TITLE
Enable CD for kommander UI

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
**What type of PR is this?**

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by armandgrillet — APPROVED** [review]
LGTM.

File: stable/kommander/requirements.lock [hunk 1]
```diff
@@ -10,18 +10,18 @@ dependencies:
   version: 0.1.15
 - name: kommander-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.8.0
+  version: v0.8.0
```
Comment (hectorj2f) [inline stable/kommander/requirements.lock:13]: this is weird, which helm version are you using ?

File: stable/kommander/values.yaml [hunk 2]
```diff
@@ -197,6 +197,7 @@ grafana:
 kommander-ui:
   showCost: true
   impersonateUser: true
+  showCD: true
```
Comment (hectorj2f) [inline stable/kommander/values.yaml:200]: what is `showCD` for ?

**Review by juliangieseke — APPROVED** [review]

File: stable/kommander/values.yaml (hunk 2 above)
Comment (TattdCodeMonkey) [reply stable/kommander/values.yaml:200]: It's a feature flag for turning on Continuous Deployment in Kommander.

**Pull request merged**